Makefile

```diff
@@ -152,6 +152,7 @@ fixtures: fixtures/docker \
 	fixtures/file-perf \
 	fixtures/file-mixed \
 	fixtures/python-pypi \
+	fixtures/rpm-manually-created-fixtures \
```

Comment:
I would just create a new fixture called "rpm-kickstart" and uncomment/update "rpm-with-non-utf-8". I imagine that someone might want to just generate one or the other but not both.